db/structure.sql

```diff
@@ -3010,8 +3008,6 @@ INSERT INTO "schema_migrations" (version) VALUES
 ('20220613212850'),
 ('20220615155458'),
 ('20220621144511'),
-('20220714144937'),
-('20220822134957'),
-('20220914124822');
+('20220714144937');
```

Comment:
Yeah, probably not good that migrations have been removed.  
```suggestion
('20220714144937'),
('20220822134957'),
('20220914124822');
```
For this PR, I think that should be sufficient.

Reply:
Thanks Elliot, I committed your suggestion